src/Cell.cpp

```diff
@@ -1,70 +1,64 @@
 
-
 #include "TMath.h"
 #ifdef STANDALONE
 #include "Cell.h"
 #else
 #include "HGCalSimulation/FastShower/interface/Cell.h"
 #endif
 
+#include <vector>
+
+
+uint32_t Cell::id(uint16_t i, uint16_t j, uint16_t k) {
+    // id format = iiijjjkk
+    return i*100000 + j*100 + k;
```

Comment:
Clearer to use bit shifts as was done before (e.g. `id |= (j<<16)`)

Reply:
I cannot use bit with the implementation of all layers. For a given cell I need to access the layer for the display stuff and by using the bit shift I loose the information.

Reply:
Yes you can. In the exact same way as it was done before except that you have k in addition so you need to change the sharing of the 32 bits.